.github/workflows/test-run.yml

```diff
@@ -99,11 +102,10 @@ jobs:
           proxy_host: ${{ env.FRONTEND_HOST }}
           proxy_username: ${{ env.BACKEND_USER }}
           proxy_key: ${{ secrets.DEPLOY_C3_TEST }}
+          command_timeout: 30m
           script: |
             cd yaptide
-            rm -rf venv
-            python3 -m venv env
-            source env/bin/activate
-            python3 -m pip install --upgrade pip
-            pip3 install -r requirements.txt
-            FLASK_ENV=developement FLASK_APP=yaptide.application PATH=$PATH:${{ env.SHIELDHIT_PATH }} screen -d -m flask run --host 0.0.0.0
+            cp -r /home/ubuntu/usr/shieldhit ./shieldhit/
```

Comment:
Can you copy single executable instead of directory ?
The same to copying inside the docker

Reply:
```suggestion
            cp -r /home/ubuntu/usr/shieldhit/v0.9.2/bin/shieldhit .
```

Reply:
Yep, that's a great idea.